src/applications/managing-app-configuration.md

```diff
@@ -52,6 +52,19 @@ the application to an external PostgreSQL database.
 Detachments can be performed with `deis config:unset`.
 
 
+# Slugbuilder Cache
+By default, apps using the slugbuilder will have caching turned on. This means that deis will
```

Comment:
can you capitalize and link Slugbuilder to `../understanding-workflow/components.md#builder-builder-slugbuilder-and-dockerbuilder`? Also would you please capitalize "Deis" and add a blank newline character between the paragraph title and the body (see line 40)? Thanks